src/resources/metrics.py

```diff
@@ -0,0 +1,23 @@
+from flask_restful import Resource
+from src.model.metric import Metrics
+from src.model.pre_config import PreConfig
+from flask import request
+
+
+class PreConfigMetrics(Resource):
+    def post(self):
+
+        data = request.get_json(force=True)
+
+        pre_configuration_id = data["pre_config_id"]
+
+        try:
+            if PreConfig.objects.with_id(pre_configuration_id) is None:
+                return 404
+            else:
+                Metrics(pre_config_id=data["pre_config_id"]).save()
+                data.pop("pre_config_id", None)
+                Metrics(metrics_list=data).save()
```

Comment:
Não é bom dar 2 `save()` em seguida, isso força uma ida no banco de dados (que é um pouco lento). Que tal:

```py
Metrics(
  pre_config_id=data.pop("pre_config_id", None),
  metrics_list=data
).save()
```

Testei aqui e funcionou.